src/Microsoft.ML.KMeansClustering/KMeansPlusPlusTrainer.cs

```diff
@@ -26,7 +26,18 @@
 
 namespace Microsoft.ML.Trainers
 {
-    /// <include file='./doc.xml' path='doc/members/member[@name="KMeans++"]/*' />
+    /// <summary>
```

Comment:
Do you think scoring function to the trained modle should be explained?

Reply:
you mean Predict function? the one that predicts which cluster data point goes to? 

---
In reply to: [276525272](https://github.com/dotnet/machinelearning/pull/3387#discussion_r276525272) [](ancestors = 276525272)

Reply:
or do u mean how distance is calculated when algo converges to centroids?

---
In reply to: [277107516](https://github.com/dotnet/machinelearning/pull/3387#discussion_r277107516) [](ancestors = 277107516,276525272)

Reply:
In either case I dont think so. For former its not specific KMeans. For latter first how am I supposed to do that? Should I deduce this from the code? Second I dont think this information is important to know for .NET devs. All they need to know that this is an improved version of KMeans and high level details of improvements that are described in wiki links

---
In reply to: [277107690](https://github.com/dotnet/machinelearning/pull/3387#discussion_r277107690) [](ancestors = 277107690,277107516,276525272)

Reply:
That's what I did. Debugging ML model is debugging math equation. Without them, how can user improve their models? We keep saying that .NET devs can ignore details. However, this doesn't look super true in ML.NET's gitter channel.

Reply:
sure, once we have more time we could prioritize such tasks to the top, currently this are lower pri to the rest of things that need to be done.

---
In reply to: [277148107](https://github.com/dotnet/machinelearning/pull/3387#discussion_r277148107) [](ancestors = 277148107)